tests/unit/src/test/scala/tests/ImplementationLspSuite.scala

```diff
@@ -255,8 +254,9 @@ class ImplementationLspSuite extends BaseRangesSuite("implementation") {
        |""".stripMargin,
   )
 
+  // we currently don't collect information about overridden symbols in JavaMtags
   check(
-    "java-classes",
+    "java-classes".ignore,
```

Comment:
To make this work we need `JavaMtags` (or `JavaTopLevelMatgs`) to return information about overridden symbols. Now we don't know that `RuntimeException` implements `Exception`, and only `MyException` will be found.